src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -30,6 +32,22 @@ public class OpenIdConnectAuthenticationHandler : AuthenticationHandler<OpenIdCo
     {
         private const string NonceProperty = "N";
         private const string UriSchemeDelimiter = "://";
+
+        private const string InputTagFormat = @"<input type=""hidden"" name=""{0}"" value=""{1}"" />";
```

Comment:
@blowdart please make sure you review these codez for HTML injection / CSRF / etc.